lib/dbservice_web/router.ex

```diff
@@ -3,35 +3,36 @@ defmodule DbserviceWeb.Router do
+    post("/group-type/:id/update-sessions", GroupTypeController, :update_sessions)
 
-    resources "/user", UserController, only: [:index, :create, :update, :show]
-    post "/user/:id/update-groups", UserController, :update_groups
-    resources "/student", StudentController, except: [:new, :edit]
-    post "/student/register", StudentController, :register
-    patch "/student/register/:id", StudentController, :update_student_with_user
-    resources "/teacher", TeacherController, except: [:new, :edit]
-    post "/teacher/register", TeacherController, :register
-    patch "/teacher/register/:id", TeacherController, :update_teacher_with_user
-    resources "/school", SchoolController, except: [:new, :edit]
-    resources "/enrollment-record", EnrollmentRecordController, except: [:new, :edit]
-    resources "/session", SessionController, only: [:index, :create, :update, :show]
-    post "/session/:id/update-groups", SessionController, :update_groups
-    resources "/session-occurrence", SessionOccurenceController, except: [:new, :edit]
-    resources "/user-session", UserSessionController, except: [:new, :edit]
-    resources "/group-session", GroupSessionController, except: [:new, :edit]
-    resources "/program", ProgramController, except: [:new, :edit]
-    resources "/batch", BatchController, except: [:new, :edit]
-    resources "/batch-program", BatchProgramController, except: [:new, :edit]
-    resources "/group-type", GroupTypeController, except: [:new, :edit]
+    resources("/user", UserController, only: [:index, :create, :update, :show])
+    post("/user/:id/update-groups", UserController, :update_groups)
+    resources("/student", StudentController, except: [:new, :edit])
+    post("/student/register", StudentController, :register)
+    patch("/student/register/:id", StudentController, :update_student_with_user)
+    resources("/teacher", TeacherController, except: [:new, :edit])
+    post("/teacher/register", TeacherController, :register)
+    patch("/teacher/register/:id", TeacherController, :update_teacher_with_user)
+    resources("/school", SchoolController, except: [:new, :edit])
+    resources("/enrollment-record", EnrollmentRecordController, except: [:new, :edit])
+    resources("/session", SessionController, only: [:index, :create, :update, :show])
+    post("/session/:id/update-groups", SessionController, :update_groups)
+    resources("/session-occurrence", SessionOccurenceController, except: [:new, :edit])
+    resources("/user-session", UserSessionController, except: [:new, :edit])
+    resources("/group-session", GroupSessionController, except: [:new, :edit])
+    resources("/program", ProgramController, except: [:new, :edit])
+    resources("/batch", BatchController, except: [:new, :edit])
+    resources("/batch-program", BatchProgramController, except: [:new, :edit])
+    resources("/group-type", GroupTypeController, except: [:new, :edit])
+    resources("/form_schema", FormSchemaController)
```

Comment:
route should always be hyphenated. let's change it to
```suggestion
    resources("/form-schema", FormSchemaController)
```